Day2/exercise2/solution.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+	"time"
+)
+
+var sum = int(0)
+
+func rating(j int, wg *sync.WaitGroup) {
+	defer wg.Done()
+	sum += rand.Intn(20)
+	n := rand.Intn(5)
+	time.Sleep(time.Duration(n) * time.Millisecond)
+}
+func main() {
+	var wg sync.WaitGroup
+	for i := 0; i < 200; i++ {
```

Comment:
this 200 will be used at multiple places. hence have it in a variable so that if you want to change that number you have to change at one place only.  this will also reduce human errors like mistakenly writing 20 somewhere instead of 200.